google/cloud/bigquery/job/base.py

```diff
@@ -415,7 +415,9 @@ def __init__(self, job_id, client):
     @property
     def configuration(self) -> _JobConfig:
         """Job-type specific configurtion."""
-        configuration = self._CONFIG_CLASS()
+        # TODO: The typehinting for this needs work. Setting this pragma to temporarily
```

Comment:
similar tohttps://github.com/googleapis/python-bigquery/pull/2133/files#r1962595719

Reply:
All TODO items in this PR have been added to Issue #2132